code/controllers/subsystems/item_worths.dm

```diff
@@ -1,17 +1,77 @@
-//Negative values indicate that instances of these types should use the Value proc
-//Mainly used so that stuff inside them can also add to their value, and other things like material,
-//stuff like that.
+/var/global/datum/controller/subsystem/item_worths/SSItemWorths
 
-var/list/worths = list(
-//ROBOT ASSEMBLIES,
+/datum/controller/subsystem/item_worths
+	name = "Item Worths"
+	flags = SS_NO_FIRE
+	init_order = SS_INIT_MISC_FIRST
```

Comment:
This need to be this early?

Reply:
Yes, because other systems will require the item values (for example vendors)